src/pages/careers/_dept-layout-components/_zoho-job.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { Helmet } from 'react-helmet'
+import { OpenPositionsProps } from './_dept-layout.types'
+import RightArrow from 'images/svg/career-right-arrow.svg'
+
+type DataProps = {
+    title: OpenPositionsProps['title']
+}
+
+const ZohoJob = (data: DataProps) => {
+    const [scriptLoaded, setScriptLoaded] = useState(false)
+
+    useEffect(() => {
+        const careerScript = document.createElement('script')
+        careerScript.src =
+            'https://js.zohostatic.com/recruit/embed_careers_site/javascript/v1.0/embed_jobs.js'
+        careerScript.addEventListener('load', () => setScriptLoaded(true))
+        document.body.appendChild(careerScript)
+    }, [])
+
+    useEffect(() => {
+        if (!scriptLoaded) return
+        addScriptForZoho()
+    }, [scriptLoaded])
+
+    const addScriptForZoho = () => {
+        const addScript = (settings) => {
+            const script = document.createElement('script')
+            const { async, text, src, id } = settings
+
+            if (async) script.async = settings['async']
+            if (text) script.text = settings['text']
+            if (src) script.src = settings['src']
+            if (id) script.id = settings['id']
+            document.head.appendChild(script)
+        }
+
+        addScript({
+            text: `
+            window.rec_embed_js && window.rec_embed_js.load({
+				widget_id:"rec_job_listing_div",
+				page_name: "${data.title}",
+				source:"CareerSite",
+				site:"https://deriv.zohorecruit.com",
+				empty_job_msg:"Sorry, there are currently no open positions."
+			});`,
+        })
```

Comment:
Can we add async here?

Reply:
No, for this we're using the Zoho configuration.